src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1496,7 +1376,8 @@ void SizePIU(EnergyPlusData &state, int const PIUNum)
     }
 
     if (CurTermUnitSizingNum > 0) {
-        TermUnitSizing(CurTermUnitSizingNum).MinFlowFrac = thisPIU.MinPriAirFlowFrac;
+        TermUnitSizing(CurTermUnitSizingNum).MinPriFlowFrac = thisPIU.MinPriAirFlowFrac;
```

Comment:
`MinFlowFrac` is actually the minimum primary air flow fraction. This was changed throughout.

Reply:
I see that `MinFlowFrac` (now `MinPriAirFlowFrac`) is used in the autosizing routines as such:
https://github.com/NREL/EnergyPlus/blob/db1a2350f8db571cff545d64f97dee7acb8d8126/src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc#L115-L120

`MinPriAirFlowFrac` is the ratio of minimum primary air to maximum primary air. I think that perhaps for the sizing routine (as shown above) we would need to use the ratio of minimum primary air to maximum total air flow (primary and secondary air flow). This is probably beyond the scope of this PR.